app/assets/stylesheets/application.scss

```diff
@@ -19,6 +20,39 @@ html {
 
 body {
   height: 100%;
+  overflow: hidden;
+  display: flex;
+  flex-direction: column;
+  #mainContainer {
+    display: flex;
+    flex: 1;
+
+      .nav-pf-vertical {
+        position: static;
+        display: flex;
+        .list-group {
+              overflow-y: auto;
+              overflow-x: hidden;
+              max-height: inherit;
+              flex: 1;
+              li.list-group-item {
+                
+                a,
+                a:after {
+                  color: #efefef;
+                  &:hover {
+                    color: #fff;
+                  }
+                }
+              }
+        }
+      }
+      #main {
+        overflow: auto;
+        flex: 1;
```

Comment:
Another thing to note is that flexbox isn't fully supported on IE 10 and 11.  Not sure what this would look like in IE but it's something worth trying.